src/Umbraco.Web.UI.Client/src/views/common/dialogs/login.html

```diff
@@ -40,7 +40,10 @@
                 </div>
 
                 <div class="control-group" ng-class="{error: loginForm.password.$invalid}">
-                    <input type="password" ng-model="password" name="password" class="input-xlarge" localize="placeholder" placeholder="@placeholders_password" autocomplete="off" />
+                    <input type="{{showpassword ? 'text' : 'password'}}" ng-model="password" name="password" class="input-xlarge password" ng-class="{'has-password': password.length > 0}" localize="placeholder" placeholder="@placeholders_password" autocomplete="off" />

```

Comment:
We could check with this example http://codepen.io/gymbry/full/fJchw/ which use the same principle.. however I just noticed that the Microsoft Edge browser actually add this feature to password input fields.

![image](https://cloud.githubusercontent.com/assets/2919859/11029965/120348c8-86cb-11e5-80ac-63c8b77fac01.png)

the same for IE11 (at least on Windows 10)
![image](https://cloud.githubusercontent.com/assets/2919859/11029996/4c6f5c5e-86cb-11e5-8acd-0ca5e8b66907.png)